library/eclipse-mosquitto

```diff
@@ -1,6 +1,6 @@
 Maintainers: Roger Light <roger@atchoo.org> (@ralight)
 GitRepo: https://github.com/eclipse/mosquitto.git
-GitCommit: 1853bfc678c255367e7c2c2f138da7bf47054117
+GitCommit: 51738d41616aafc8a4b6632dfda8aaccc1d93d9f
```

Comment:
This like this was likely intended to be `da2879c33b6baf00b8b50a38da8d7f176cd85afa` instead? (`51738d41616aafc8a4b6632dfda8aaccc1d93d9f` exists, but doesn't appear to be part of any branches: https://github.com/eclipse/mosquitto/commit/51738d41616aafc8a4b6632dfda8aaccc1d93d9f)

Reply:
Yes, you're right. I remember fixing that but obviously didn't push it on to the PR. Sorry about the mixup.